src/components/LoansByCategory/ViewToggle.vue

```diff
@@ -39,51 +39,48 @@ export default {
 @import 'settings';
 
 .view-toggle {
```

Comment:
most of the changes below are b/c I nested everything inside of view-toggle, which moved it over. 


Style changes were show in text labels on large up (instead of medium up) and setting the spacing to .75 rem on either side of the dividers when the labels are hidden.